release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3244,6 +3244,11 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.11.26 --pullspecs
 ----
 
+[id="ocp-4-11-26-known-issues"]
+==== Known issues
+
+* This release regresses functionality where SR-IOV secondary network interfaces configured with static MAC address attached to pods may not provide full network connectivity. This issue only impacts SR-IOV virtual functions based on Intel Ethernet Network Adapter X710 products (i40e/iavf Linux kernel drivers). For more information, see link:https://issues.redhat.com/browse/OCPBUGS-5139[*OCPBUGS-5139*].
```

Comment:
Change "may" to "might," per [IBM standards.](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#may)